csv2sql.drush.inc

```diff
@@ -240,4 +251,42 @@ function csv2sql_get_column_name($col_name) {
   return '_' . trim(strtolower(str_replace(array('-', ' ', '/'), '_', $col_name)));
 }
 
+/**
+ * Scan a directory for csv files.
+ *
+ * Scans the directory given by the user in the command,
+ * Scans all the sub-directories of that directory,
+ * Searches for files with 'csv' extension to be added to the convert to SQL process.
+ *
+ * @param string $dir
+ *    The path of the directory provided in the command.
+ *
+ * @return array
+ *    An array of 'csv' files found in the directory.
+ */
+function csv2sql_scan_for_csv_files($dir) {
+  $csv_files = array();
+  // If the directory does not exist, print a message, stop the function.
+  if (!file_exists($dir)) {
+    drush_print('Given directory does not exist.');
+    return FALSE;
+  }
+  // Add a trailing slash and a * so the glob returns everything in the directory,
+  // Trim trailing slash and add them here, To make sure we never have '//'.
+  $items_in_dir = glob(rtrim($dir, '/') . DIRECTORY_SEPARATOR . '*');
+
+  foreach ($items_in_dir as $item) {
+    // If current item is a 'csv' file, Add to the global variable.
```

Comment:
`// If current item is a 'csv' file, Add to the global variable.` => `// If current item is a 'csv' file, add it to the list of files.`